File: problemas/iniciante/1541/1541.cs
using System;
using System.Collections.Generic;
using System.Linq;

class URI
{
    static void Main(string[] args)
    {
        string entrada;

        while ((entrada = Console.ReadLine()) != null)
        {
            List<int> numeros = entrada.Trim().Split(' ').Select((x) => int.Parse(x)).ToList();
            if (numeros.Count == 1)
            {
                break;
            }

            Console.WriteLine($"{Math.Floor(Math.Sqrt(numeros[0] * numeros[1] * 100.0 / numeros[2])):0}");
        }
    }
}

File: problemas/iniciante/1541/1541.c
#include <stdio.h>
#include <math.h>

int main()
{
    int A, B, C;

    while (scanf("%d", &A) != EOF)
    {
        if (A == 0)
        {
            break;
        }

        scanf("%d %d", &B, &C);
        printf("%.0lf\n", floor(sqrt(A * B * 100.0 / C)));
    }

    return 0;
}